src/components/Dashboard/widgets/TokenDetailWidget.tsx

```diff
@@ -125,13 +131,28 @@ const TokenDetail: React.FC = () => {
             <MetadataContent>
               <Description>{tokenDetail.description}</Description>
               {tokenDetail.dataAsJson?.url && (
-                <ContentLink
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  href={tokenDetail.dataAsJson?.url}
-                >
-                  {tokenDetail.dataAsJson?.url}
-                </ContentLink>
+                <>
+                  <ContentLink
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    href={tokenDetail.dataAsJson?.url}
+                  >
+                    {tokenDetail.dataAsJson?.url}
+                  </ContentLink>
+                  {tokenDetail.dataAsJson?.url.match(EXTRACT_IPFS_HASH_REGEX) && (
+                    <OpenInGatewayButton theme={Colors.TRANSPARENT}>
+                      <a
+                        style={{ color: 'white' }}
+                        href={tokenDetail.dataAsJson?.url}
```

Comment:
this will open the link as normal, instead of opening it in a gateway. eg.: `ifs://<hash>` will not work in the browser. You need to check if it is an ipfs native link and transform into a `http://<gatway>/<hash>` link.

There's a `DEFAULT_IPFS_FALLBACK_GATEWAY` in the `defines.ts` file